app/actions/definitions/documents.tsx

```diff
@@ -828,6 +830,26 @@ export const permanentlyDeleteDocument = createAction({
   },
 });
 
+export const permanentlyDeleteDocumentsInTrash = createAction({
+  name: ({ t }) => t("Empty"),
+  analyticsName: "Empty trash",
+  section: TrashSection,
+  icon: <TrashIcon />,
+  dangerous: true,
+  visible: ({ stores }) => stores.documents.deleted.length > 0,
```

Comment:
I realized one thing here, the route is admin only – so this needs to be visible to admins only too, can check `user.isAdmin`